docs/writing-docs/introduction.md

```diff
@@ -19,4 +19,4 @@ You can also create free-form pages for each component using [MDX](./mdx.md), a
 
 In both cases, you’ll use [Doc Blocks](./doc-blocks.md) as the building blocks to create full featured documentation.
 
-Docs is autoconfigured to work out of the box in most use cases. In some cases you may need or want to tweak the configuration. Read more about it in the [advanced readme](../../addons/docs/ADVANCED-README.md).
+Docs is autoconfigured to work out of the box in most use cases. In some cases you may need or want to tweak the configuration. Read more about it in the [advanced readme](../../addons/docs/README.md).
```

Comment:
@pascalduez can you revert this and leave the advanced readme link? I know that is a bit cumbersome to lead to a 404 but it's being taken care of. But it didn't make it into the repo.

Reply:
Oh, sure.